0xProto.glyphs

```diff
@@ -1,6 +1,13 @@
 {
 .appVersion = "3151";
 .formatVersion = 3;
+classes = (
+{
+automatic = 1;
+code = "A Aacute Acircumflex Adieresis Agrave Aring Atilde AE B C Ccedilla D Eth E Eacute Ecircumflex Edieresis Egrave F G H I Iacute Icircumflex Idieresis Igrave J K L M N Ntilde O Oacute Ocircumflex Odieresis Ograve Oslash Otilde OE P Q R S Scaron T U Uacute Ucircumflex Udieresis Ugrave V W X Y Yacute Ydieresis Z Zcaron";
+name = Uppercase;
+}
+);
```

Comment:
This also seems to be auto-generated code, but why is this in a class...?

Reply:
The class is used in the `ccmp` feature for the `sub @Uppercase @MarksComb' by @MarksCombCase;`. This subsitutes the combining marks by their uppercase variants when combined with an uppercase. For example X̌ would start as `X caroncomb` and be substituted by `X caromcomb.case` after this lookup.

Reply:
Thanks. I was wondering if the `@CombiningTopAccents` and `@Markscomb` classes are defined inside the lookup block, whereas `@Uppercase` is defined outside it?

Reply:
I think Glyphs.app may use @Uppercase in other features.

Reply:
Yeah, you're right. It looks like `@Uppercase` is used for things beyond just the ccmp features.